WeGa/Utils.cs

```diff
@@ -17,5 +20,25 @@ public static double getScreenHeight()
         {
             return System.Windows.SystemParameters.PrimaryScreenHeight;
         }
+
+        //Method to get shuffled letters.
+        public static string getLetters() {
+            Random rnd = new Random();
+            char c;
+            string letters = "";
+            do
+            {
+                c = vowelSet[rnd.Next(0, 3)];
```

Comment:
shouldn't this be `c = vowelSet[rnd.Next(0, 5)];` or `c = vowelSet[rnd.Next(0, 4)];`